clouddriver-web/config/clouddriver.yml

```diff
@@ -266,6 +266,12 @@ cf:
       artifactUsername: ${cf.repo.username}
       artifactPassword: ${cf.repo.password}
 
+oraclebmcs:
+  enabled: ${ORACLEBMCS_ENABLED:false}
+  accounts:
```

Comment:
This config file is a bit of a mixed bag, but I'd recommend not adding any accounts here since it could lead to spring-config confusion down the road. Arrays across multiple spring profiles are overwritten entirely instead of merged, and it's not always obvious what the profile-load-order is. The other provider (minus CF) use this file to define provider-wide defaults, which are a little easier to reason about and change less frequently than the set of enabled accounts.